config.js

```diff
@@ -265,6 +265,11 @@ function parseArgs(options, raw_args) {
         help: 'Shorthand for "--keep-open --devtools-preserve --forward-console"',
         action: 'storeTrue',
     });
+    puppeteer_group.addArgument(['--default-timeout'], {
+        help: 'Default timeout value for various browser functions',
```

Comment:
Instead of defaulting below, can't we add `defaultValue: 30000` here and document in the help string that it's 30s?

Reply:
Sounds good to me! Made the change 👍